src/listmode_buildblock/CListModeDataPENN.cxx

```diff
@@ -0,0 +1,132 @@
+/*
+    Copyright (C) 2020-2022 University of Pennsylvania
+    This file is part of STIR.
+
+    SPDX-License-Identifier: Apache-2.0 AND License-ref-PARAPET-license
```

Comment:
probably no PARAPET license, unless you copied a lot of code. (also elsewhere)

Reply:
I think fixed all of them.